README_ar.md

```diff
@@ -0,0 +1,602 @@
+
+النموذج نفسه هو [وحدة `nn.Module` من Pytorch](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) أو [`tf.keras.Model` من TensorFlow](https://www.tensorflow.org/api_docs/python/tf/keras/Model) (حسب النظام الأساسي الذي تستخدمه) والذي يمكنك استخدامه كالمعتاد. [يشرح هذا الدليل](https://huggingface.co/docs/transformers/training) كيفية دمج مثل هذا النموذج في حلقة تدريب كلاسيكية لـ PyTorch أو TensorFlow، أو كيفية استخدام واجهة برمجة التطبيقات `Trainer` لدينا للضبط الدقيق بسرعة على مجموعة بيانات جديدة.
+
+## لماذا يجب علي استخدام Transformers؟
+
+1. نماذج حديثة سهلة الاستخدام:
+    - أداء عالي في فهم وتوليد اللغة الطبيعية، الرؤية الحاسوبية، ومهام الصوت.
+    - حاجز منخفض للدخول للمعلمين والممارسين.
+    - عدد قليل من التجريدات التي تواجه المستخدم مع ثلاث فئات فقط للتعلم.
+    - واجهة برمجة تطبيقات موحدة لاستخدام جميع نماذجنا المدربة مسبقًا.
+
+2. تكاليف حوسبة أقل، بصمة كربون أصغر:
+    - يمكن للباحثين مشاركة النماذج المدربة بدلاً من إعادة التدريب دائمًا.
+    - يمكن للممارسين تقليل وقت الحوسبة وتكاليف الإنتاج.
+    - عشرات الهندسات مع أكثر من 400,000 نموذج مدرب مسبقًا عبر جميع الوسائط.
+
+3. اختيار الإطار الصحيح لكل جزء من عمر النموذج:
+    - تدريب نماذج حديثة في 3 أسطر من الكود.
+    - نقل نموذج واحد بين إطارات العمل TF2.0/PyTorch/JAX بحرية.
+    - اختيار الإطار الصحيح بسلاسة للتدريب، التقييم، والإنتاج.
+
+4. تخصيص نموذج أو مثال لاحتياجاتك بسهولة:
+    - نوفر أمثلة لكل هندسة لإعادة إنتاج النتائج التي نشرها مؤلفوها الأصليون.
+    - تتم تعرض دواخل النموذج بشكل متسق قدر الإمكان.
+    - يمكن استخدام ملفات النموذج بشكل مستقل عن المكتبة للتجارب السريعة.
+    
+## لماذا لايجب علي استخدام Transformers؟
+
+- هذه المكتبة ليست صندوق أدوات معياري من الكتل البنائية للشبكات العصبية. الكود الموجود في ملفات النموذج ليس معاد تنظيمه مع تجريدات إضافية عن عمد، حتى يتمكن الباحثون من التكرار بسرعة على كل من النماذج دون الغوص في تجريدات/ملفات إضافية.
+- واجهة برمجة التطبيقات للتدريب ليست مخصصة للعمل مع أي نموذج ولكنها مُحسنة للعمل مع النماذج التي توفرها المكتبة. لأجل حلقات التعلم الآلي العامة، يجب استخدام مكتبة أخرى (ربما، [Accelerate](https://huggingface.co/docs/accelerate)).
+- بينما نسعى لتقديم أكبر عدد ممكن من حالات الاستخدام، السكربتات في [مجلد الأمثلة](https://github.com/huggingface/transformers/tree/main/examples) لدينا هي مجرد ذلك: أمثلة. من المتوقع أنها لن تعمل مباشرةً على مشكلتك الخاصة وأنه سيتطلب منك تغيير بضعة أسطر من الكود لتكييفها مع احتياجاتك.
+
+## التثبيت
+
+### بواسطة pip
+
+تم اختبار هذا الريبو على +Python 3.8+، Flax 0.4.1+، PyTorch 1.11، و +TensorFlow 2.6.
+
+يجب تثبيت 🤗 Transformers في [بيئة افتراضية](https://docs.python.org/3/library/venv.html). إذا كنت غير مألوف ببيئات Python الافتراضية، راجع [دليل المستخدم](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/).
```

Comment:
Link to the Python virtual environment guide won't work

Reply:
![screenshot_2024-02-20-212226](https://github.com/huggingface/transformers/assets/71812454/8bd10670-71af-49df-ba15-852ebb786b52)